hw/drivers/display/ssd1673/syscfg.yml

```diff
@@ -0,0 +1,47 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+syscfg.defs:
+    SSD1673_OS_DEV_NAME:
+        description: 'os_dev device name for SSD1673'
+        value: '"ssd1673"'
+
+    SSD1673_BUSY_PIN:
+        description: 'BUSY pin for SSD1673'
+        value: 14
```

Comment:
I think it would be better to change value:14 to
```
        value:
        restrictions:
            - $notnull
```
and define pin in bsp instead, since those values do not make sense in general.
Same applies to other pins.